Restfull API/api/app/auth/__init__.py

```diff
@@ -0,0 +1,5 @@
+from flask import Blueprint
+
+auth = Blueprint('auth', __name__)
```

Comment:
What is a blueprint?

Reply:
A blueprint is more like packages for the different sections of my app. Am using it to separate the different features or sections of my app example, authentication is one section, business will be another...so on so forth.